views/index.py

```diff
@@ -324,6 +324,28 @@ def reserve(copy_id):
             abort(500)
     return redirect(url_for('library.index'))
 
+ '''That doesn't working:
+ 
+ res_end = RentalLog._reservation_begin.replace(tzinfo=pytz.utc)\
+ 					.astimezone(tz=pytz.timezone('Europe/Warsaw'))
+'''
+
+@library.route('/check_reservation_status_db')
+def check_reservation_status_db():
+    Reserved_list = db.session.query(RentalLog)\
+        .filter(RentalLog.book_status == BookStatus.RESERVED)\
+        .filter(RentalLog._reservation_end < datetime.utcnow)\
+        .all()
+    for reserved_row in Reserved_list:
```

Comment:
Could we eliminate that for loop and handle that in one query?

Reply:
That was my proposal!, but since we have more then one book status in different two tables in our DB, there is impossible to use one query.
That was Janusz proposal with for